core/src/main/scripts/importer/validateData.py

```diff
@@ -3065,6 +3068,14 @@ def validate_study(study_dir, portal_instance, logger, relaxed_mode):
     logger.info('Validation complete')
 
 
+def get_pom_path():
+    """
+    Get location of pom.xml. In system and integration test this is mocked.
+    """
+    pom_path = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__)))))) + "/pom.xml"
```

Comment:
Nice, it seems like this also just ‘bottoms out’ without errors even if the script path is the root directory. But please do canonicalise the script's path using `os.path.realpath()` here, or it will break situations where the script is installed into `$PATH` via a symlink, such as the Dockerised setups.